priv/repo/migrations/20170710175356_create_email.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Askbywho.Repo.Migrations.CreateEmail do
```

Comment:
Mmm, the uniqueness of emails would make sense so you don't have multiple entries for the same email address, however, if we added the timestamps to the linking table, then if the same user submitted multiple times, we'd be able to see the submissions and the time of those submissions.  Thus, the "brands_emails" table really is a submissions history, and then the "emails" and "brands" table is really a unique list of names.